addons/project_mrp/tests/test_project_mrp.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+from odoo.tests import tagged
+from odoo import Command
+
+@tagged('-at_install', 'post_install')
+class TestProjectMrp(TestProjectCommon):    
+    def test_project_profitability_with_manufacturing_orders(self):
+        #  When the analytic account has some manufacturing orders , they should be counted only once
+        mrp_account = self.env['ir.module.module']._get('mrp_account')
+        if mrp_account.state != 'installed':
+            self.skipTest("mrp_account module is not installed")
+        project_account = self.env['ir.module.module']._get('project_account')
+        if project_account.state != 'installed':
+            self.skipTest("project_account module is not installed")
```

Comment:
In fact, I suggest you to move the test in `project_account` to avoid doing that and be sure we have the modules needed.

Here, since `mrp_account` is in the dependencies of `project_mrp` module then we can be sure `mrp_account` will be installed.

`project_account` module will be installed if project and account modules are installed and it would also be the case since `project` will be installed when `project_mrp` and `account` will also be installed thanks to `stock_account` module in the dependencies of `mrp_account`. And so it means `project_account` will also be installed, and thus those checks are useless. 🙂

Reply:
I understand , thanks for the explanation , I will removed them.